src/Raven.Server/Config/Categories/StorageConfiguration.cs

```diff
@@ -40,10 +40,9 @@ public class StorageConfiguration : ConfigurationCategory
         public int MaxConcurrentFlushes { get; set; }
 
         [Description("Time to sync after flash in seconds")]
-        [DefaultValue(30)]
+        [DefaultValue(120)]
         [TimeUnit(TimeUnit.Seconds)]
-        [ConfigurationEntry("Storage.TimeToSyncAfterFlashInSec", ConfigurationEntryScope.ServerWideOrPerDatabase)]
-        [ConfigurationEntry("Storage.TimeToSyncAfterFlushInSec", ConfigurationEntryScope.ServerWideOrPerDatabase)]
```

Comment:
We had 2 separate option on purpose. Probably someone make a typo 'flash' instead of 'flush' initially so we fixed it by adding _another_ entry here. But we can't remove the previous one because of backward compatibility.

Reply:
Ahh OK